src/pages/community/getting-help.mdx

```diff
@@ -6,33 +6,61 @@ How to engage with the Pants community.
 
 The [Pants community](/community/members) is friendly, welcoming, and passionate about improving the craft of software engineering. If you need help with anything Pants-related, you've come to the right place. We'd love to hear from you!
 
-## Slack
+## How to get help
 
-The best place to become acquainted with Pants community members, exchange informal chat and ask quick questions is at the project's Slack. You can join using [this link](https://docs.google.com/forms/d/e/1FAIpQLSf9zgf-MXRnVDJbrVEST3urqneq7LCcy0zw8qU-GH4hPMn52A/viewform?usp=sf_link).
+Pants community members will do their best to answer any questions that are posed in a respectful way. General questions about Pants,
+its features, language support, roadmap etc. are always welcome.
+
+If you need help debugging a specific problem you're encountering in your repo, then we ask that you help us to help you by adhering to the following:
+
+- First, do your best to solve the issue on your own. The Pants [source code](https://github.com/pantsbuild/pants), [issue tracker](https://github.com/pantsbuild/pants/issues), and [documentation](https://pantsbuild.org)
+  are useful resources for this.
+- If you're still stuck, and would like a Pants contributor to look at your problem, please provide a small public GitHub repo that reproduces the problem.
+  The README.md in this repo should provide clear instructions on how to reproduce the problem, and the architecture and OS on which it can be reproduced.
+  Posting code and config snippets to Slack is not a good alternative, as we won't know if they contain enough information to reproduce the problem.
+  A self-contained repo is far more useful.
+- Then [file an issue](https://github.com/pantsbuild/pants/issues/new/choose) that states the problem and the steps you've taken to attempt to solve it, and a link to the reproduction repo. If you find
+  an existing issue that seems relevant, please link to it, or post directly to that existing issue if you're reasonably sure it describes the same problem.
+- Then, feel free to reach out in the #general channel in Slack with a link to your issue, to kickstart the discussion.
+
+Keep in mind that this community support is provided _voluntarily_. When you ask for help you're asking busy people to devote their valuable time to solving your problem.
+You are far more likely to get a positive response if you demonstrate that you've put in the basic legwork first.
+
+As an open-source project, we always appreciate a willingness to dive in and fix issues yourself, with our guidance.
```

Comment:
We could potentially also call out "provide support to others" as a useful activity, in addition to fixing issues.